src/lib/containers/VisibilityObserver.tsx

```diff
@@ -2,7 +2,7 @@
  * This component has the option to provide an outOfView renderer, which might be useful in the future.
  */
 import * as React from 'react'
-import Observer from 'react-intersection-observer'
+import { InView } from 'react-intersection-observer'
```

Comment:
`react-intersection-observer` v9 removed the default export. The component must now be accessed via `InView`